File: MotivApp/app/src/main/java/com/motiv/piotr/DashboardActivityContract.java
package com.motiv.piotr;


public interface DashboardActivityContract {

    public interface View {

        public void navigationControllerstartDestinationById(int arg0);

        public void navigationControllerstartUserDetailActivity(com.motiv.piotr.User arg0);

        public com.motiv.piotr.User getloggedUser();
    }

    public interface Presenter {

        public void eloonNavigationItemSelected(android.view.MenuItem argument0);

        public void eloonClick(android.view.View argument0);
    }
}

File: MotivApp/app/src/main/java/com/motiv/piotr/DashboardActivityContract.kt
 
package com.motiv.piotr
import androidx.fragment.app.*
import dagger.*
import dagger.android.*
import dagger.android.support.*
import javax.inject.*

public interface DashboardActivityContract {

    public interface View {

        public fun navigationControllerstartDestinationById(arg0: Int): Unit
        public fun navigationControllerstartUserDetailActivity(arg0: com.motiv.piotr.User): Unit
        public fun getloggedUser(): com.motiv.piotr.User
    }

    public interface Presenter {

        public fun eloonNavigationItemSelected(argument0: android.view.MenuItem): Unit
        public fun eloonClick(argument0: android.view.View): Unit
    }
}
